shell/weights/process_weights.sh

```diff
@@ -0,0 +1,22 @@
+if [ "$#" -ne 4 ]; then
```

Comment:
how hard would it be to add a JSON file based on the TensorBoard logs to `weights/densenet/classification/history.json'?

Reply:
I've added a separate python script to perform this extraction and update the JSON file. Since it's logically separate from the weights processing, I'm not referencing it from this shell script.

At this point in time, it's only extracting validation accuracy. But it's a starting point from which to add other metrics such as commit author, number of epochs trained, etc.